lib/instance-detail-helper.js

```diff
@@ -20,6 +20,13 @@ const {
 
 const debug = require('debug')('mongodb-data-service:instance-detail-helper');
 
+function getReadPreferenceOptions(db) {
```

Comment:
Isn't `db.s.readPreference` a private method in the driver?

Also .. I think here we can just hardcode `ReadPreference.PRIMARY_PREFERRED`, i see no harm, may be missing something

Reply:
Yes, wrote this quick, I'll see if there's a public method we can use. Looks like we don't pass our connection model through these instance detail helpers.